source/2020-05-29-the-ember-times-issue-150.md

```diff
@@ -14,14 +14,28 @@ READMORE
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## ["My Experience with Ember.js" Part 2 🎬](https://www.youtube.com/watch?v=R2JdP4lb5Xw)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+[Cal Woolgar (@callam_woolgar)](https://twitter.com/callam_woolgar) has a second short video out with his new series, [Creating an Ember Application!](https://www.youtube.com/watch?v=R2JdP4lb5Xw) He gets into a little more detail in this video and demonstrates how easy it is to install and get started with Ember.js and it's cli.
```

Comment:
(Rhetoric questions to myself) Is "Creating an Ember Application" the name of the video or of the new series? "More detail" about what?

I thought the quoted parts were a bit ambiguous. I think these sentences can be more clear if we write,
```suggestion
[Cal Woolgar (@callam_woolgar)](https://twitter.com/callam_woolgar) has a second short video out, called [Creating an Ember Application](https://www.youtube.com/watch?v=R2JdP4lb5Xw)! He demonstrates how the Ember CLI makes it easy to install and get started with Ember.js.
```